docs/modules/operation/pages/deep-dive/admin/configuration/data-collection.adoc

```diff
@@ -38,7 +38,20 @@ Usage statistics include the following data points:
 * Number of queued updates completed.
 * Number of virtual appliances, Mini appliances, and 1U appliances that are connected to the system.
 
+For more details, navigate to the Help Improve OpenNMS dialog:
```

Comment:
```suggestion
For more details, navigate to the Help Improve OpenNMS screen:
```

Reply:
I think I'd like to keep this as "dialog," since it presents itself as an overlay on top of the Settings page. 🤔

Reply:
Of course! I didn't think of the context.